src/main/java/com/cinchapi/common/collect/Sequences.java

```diff
@@ -60,6 +66,19 @@ public static boolean isSequence(Object object) {
         return isSequenceType(object.getClass());
     }
 
+    public static <T, R> List<R> flatMap(Object sequence, Function<T, List<R>> f) {
```

Comment:
@Tylerwbrown Why have these map methods? It's better to just transform the Sequence into a stream and leverage the pipeline from there. That way we don't have to maintain a lot of duplicate functionality.

Reply:
Mainly to reduce the code complexity at the call site. We have `forEach` so I figured it was fine to add `map` and `flatMap` since they pretty much do the same thing (iterate over the list) but are more functional.

Reply:
Since `forEach` is also on `Stream` I figured that wasn't an issue.